### PR TITLE
BUILD-2322 Migrate Cirrus CI on AWS

### PR DESCRIPTION
**Pull request by elrzn**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -1,10 +1,5 @@
-gcp_credentials: ENCRYPTED[!149d4005ecdba4cdd78bb5ba22756ebb98bf8e3367ee2e9ab08c5a1608c0d3e3b501904b67a1d67c0b63085e469d7dde!]
- 
 env:
   CIRRUS_CLONE_DEPTH: 20
-  CIRRUS_VAULT_AUTH_PATH: jwt-cirrusci
```
Comment (vlad-sonar) [inline .cirrus.yml:5]: DTTO the sonar-lits
seems you are going the extra mile here, which might be good, but I'm not clear if this is something missed during migration to vault, or we have a gap in doc regarding migration to AWS

Comment (elrzn) [reply .cirrus.yml:5]: It's orthogonal to the AWS migration. I decided to do it the Cirrus modules way because we have documented it for other developers to use.

In hindsight, I should have made this change in a separate commit to make it easier to review.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -5,9 +5,6 @@ on:
     types:
       - published
 
-env:
-  PYTHONUNBUFFERED: 1
-
```
Comment (Godin) [inline .github/workflows/release.yml:10]: This change is unrelated to BUILD-2322, next time will be better to do it via separate PR.

**Review by Godin — APPROVED** [review]
